module_utils/network_lsr/argument_validator.py

```diff
@@ -347,7 +347,7 @@ def _validate_impl(self, value, name):
                 raise ValidationError(name, "invalid key '%s'" % (setting))
             if isinstance(validator, ArgValidatorDeprecated):
                 setting = validator.deprecated_by
-                validator = self.nested.get(setting, None)
```

Comment:
There is something wrong in the commit message: `"validator.deprecated_by" is not use input.` not sure what you mean exactly.

Also, it would be nice to keep the summary short enough that it is not wrapped.